libass/ass_utils.c

```diff
@@ -504,7 +504,8 @@ int lookup_style(ASS_Track *track, char *name)
     i = track->default_style;
     ass_msg(track->library, MSGL_WARN,
             "[%p]: Warning: no style named '%s' found, using '%s'",
-            track, name, track->styles[i].Name);
+            track, name ? name : "(null)",
+            track->styles[i].Name ? track->styles[i].Name : "(null)");
```

Comment:
Can default style have `NULL` name? Anyway, I think that error handling should be uniform: better to skip reporting here too.

Reply:
Yes, if there's a [strdup failure](https://github.com/libass/libass/blob/master/libass/ass.c#L244) for libass' (not the script's) default style, there will be a style with `NULL` name.
iirc there are multiple cases that assume there's at least (this) one style – and since checks for `NULL` font names are needed anyway and this is the only place where the *style* name matters, checking for `NULL` here is less intrusive than rewriting the code to not add the default style if a `strdup` failed and to deal with no styles to choose from.

Message is now skipped if style `Name` is `NULL.